templates/chat.mustache

```diff
@@ -69,7 +82,7 @@
       })
       .then(({token}) => token)
   ]);
-
+  const completedactivities = [];
```

Comment:
```suggestion
```
Isn't the `completedActivities` variable already defined in the Mustache template?

Reply:
`completedactivities` is defined in the mustache template, but as an empty array. I'll fix that using the data passed from the block. I'll fix that.